neuromation/api/config.py

```diff
@@ -33,25 +36,46 @@ def from_config(cls, data: Dict[str, Any]) -> "_PyPIVersion":
             certifi_pypi_version = pkg_resources.parse_version(
                 data["certifi_pypi_version"]
             )
+            upload_time_str = data.get("certifi_pypi_upload_date")
+            certifi_pypi_upload_date = (
+                cls._deserialize_date(upload_time_str) if upload_time_str else date.min
+            )
             certifi_check_timestamp = int(data["certifi_check_timestamp"])
         except (KeyError, TypeError, ValueError):
             # config has invalid/missing data, ignore it
             certifi_pypi_version = cls.NO_VERSION
             certifi_check_timestamp = 0
+            certifi_pypi_upload_date = date.min
         return cls(
             pypi_version=pypi_version,
             check_timestamp=check_timestamp,
             certifi_pypi_version=certifi_pypi_version,
+            certifi_pypi_upload_date=certifi_pypi_upload_date,
             certifi_check_timestamp=certifi_check_timestamp,
         )
 
     def to_config(self) -> Dict[str, Any]:
-        return {
+        res = {
             "pypi_version": str(self.pypi_version),
             "check_timestamp": int(self.check_timestamp),
             "certifi_pypi_version": str(self.certifi_pypi_version),
             "certifi_check_timestamp": int(self.certifi_check_timestamp),
         }
+        if self.certifi_pypi_upload_date != date.min:
+            value = self._serialize_date(self.certifi_pypi_upload_date)
+            res["certifi_pypi_upload_date"] = value
```

Comment:
note: since certifi uses [calver](https://calver.org), there will be a duplicating information in `.nmrc`:
```
pypi:
  certifi_pypi_upload_date: '2019-06-16'
  certifi_pypi_version: 2019.6.16
  ...
```